.changeset/convert-graph-to-esm-package.md

```diff
@@ -0,0 +1,6 @@
+---
+"@blockprotocol/graph": patch
+---
+
+convert to esm package
+add readonly handler to GraphEmbedderHandler
```

Comment:
0.0.11 hasn't been published, will this bump it again to 0.0.12? if so we should delete this. if it doesn't then that's all good

Reply:
Removed in [9b074b5](https://github.com/blockprotocol/blockprotocol/pull/482/commits/9b074b56b6fb6e9d2757febb8fbe404b5da9c2dd)